x/multisig/decorator.go

```diff
@@ -74,7 +74,7 @@ func (d Decorator) authMultisig(ctx weave.Context, store weave.KVStore, tx weave
 		var power Weight
 		for _, p := range contract.Participants {
 			if d.auth.HasAddress(ctx, p.Signature) {
-				power += p.Power
+				power += p.Weight
```

Comment:
Probably best to change the local variable name to weight as well... otherwise confusing

Reply:
Yep, done it in the first iteration but then reverted :/